Fix test failed, closed channel write panic

https://travis-ci.org/skycoin/teller/builds/295600020

```
The command "go test ./cmd/..." exited with 0.
13.76s$ go test ./src/...
ok  	github.com/skycoin/teller/src/addrs	0.423s
?   	github.com/skycoin/teller/src/config	[no test files]
ok  	github.com/skycoin/teller/src/exchange	7.148s
ok  	github.com/skycoin/teller/src/monitor	1.016s
[2017-11-01T04:47:49Z] INFO btc.go:76 scanner.btc: Start bitcoin blockchain scan service config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2}
[2017-11-01T04:47:49Z] INFO btc.go:228 scanner.btc: Loading unprocessed deposit values
[2017-11-01T04:47:49Z] INFO btc.go:236 scanner.btc: Loaded unprocessed deposit values depositsLen=0
[2017-11-01T04:47:49Z] INFO btc.go:104 scanner.btc: Begin scanning blockchain initialHash="000000000000018d8ece83a004c5a919210d67798d13aa901c4d07f8bf87b719" initialHeight=235205
[2017-11-01T04:47:49Z] DEBUG btc.go:296 scanner.btc: Scanning block hash="000000000000018d8ece83a004c5a919210d67798d13aa901c4d07f8bf87b719" height=235205
[2017-11-01T04:47:49Z] INFO btc.go:76 scanner.btc: Start bitcoin blockchain scan service config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:0}
[2017-11-01T04:47:49Z] INFO btc.go:228 scanner.btc: Loading unprocessed deposit values
[2017-11-01T04:47:49Z] INFO btc.go:236 scanner.btc: Loaded unprocessed deposit values depositsLen=0
[2017-11-01T04:47:49Z] INFO btc.go:305 scanner.btc: Counted deposits from block hash="000000000000018d8ece83a004c5a919210d67798d13aa901c4d07f8bf87b719" height=235205 scannedDeposits=31
[2017-11-01T04:47:49Z] INFO btc.go:179 scanner.btc: Scanned deposits from block bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000018d8ece83a004c5a919210d67798d13aa901c4d07f8bf87b719" height=235205 scannedDeposits=31 totalScannedDeposits=31
[2017-11-01T04:47:49Z] DEBUG btc.go:358 scanner.btc: Waiting for the next block blockHash="000000000000018d8ece83a004c5a919210d67798d13aa901c4d07f8bf87b719" blockHeight=235205
[2017-11-01T04:47:49Z] INFO btc.go:257 scanner.btc: Sending deposit to depositC deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:2952696 Height:235205 Tx:2bc199bd35e3e49cac62cb4b96357a7570ed183fbbfe5f3fae28a40785c86032 N:1 Processed:false}
[2017-11-01T04:47:49Z] INFO btc.go:104 scanner.btc: Begin scanning blockchain initialHash="000000000000018d8ece83a004c5a919210d67798d13aa901c4d07f8bf87b719" initialHeight=235205
[2017-11-01T04:47:49Z] DEBUG btc.go:296 scanner.btc: Scanning block hash="000000000000018d8ece83a004c5a919210d67798d13aa901c4d07f8bf87b719" height=235205
[2017-11-01T04:47:49Z] INFO btc.go:275 scanner.btc: Deposit is processed deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:2952696 Height:235205 Tx:2bc199bd35e3e49cac62cb4b96357a7570ed183fbbfe5f3fae28a40785c86032 N:1 Processed:false}
[2017-11-01T04:47:49Z] INFO btc.go:257 scanner.btc: Sending deposit to depositC deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:5430 Height:235205 Tx:c15c075a3a2487c39952b6f96f694dc679b1a5922c6d483ac0eea18a968b33c4 N:0 Processed:false}
[2017-11-01T04:47:49Z] DEBUG btc.go:296 scanner.btc: Scanning block hash="00000000000001588e7832fd94d683e477a9bb25c10921d25749f93c265493f6" height=235206
[2017-11-01T04:47:49Z] INFO btc.go:275 scanner.btc: Deposit is processed deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:5430 Height:235205 Tx:c15c075a3a2487c39952b6f96f694dc679b1a5922c6d483ac0eea18a968b33c4 N:0 Processed:false}
[2017-11-01T04:47:49Z] INFO btc.go:257 scanner.btc: Sending deposit to depositC deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:86350000 Height:235205 Tx:275a1bc55ebe718e1f8b1ba5e107898fb7d6eebbc557372a96c533258486cab0 N:1 Processed:false}
[2017-11-01T04:47:49Z] INFO btc.go:275 scanner.btc: Deposit is processed deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:86350000 Height:235205 Tx:275a1bc55ebe718e1f8b1ba5e107898fb7d6eebbc557372a96c533258486cab0 N:1 Processed:false}
[2017-11-01T04:47:49Z] INFO btc.go:257 scanner.btc: Sending deposit to depositC deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:5430 Height:235205 Tx:6917609f808ee48eb34fc3cd65289b00cb4f258daa2c6cfcdb7953ed2dff712f N:0 Processed:false}
[2017-11-01T04:47:49Z] INFO btc.go:305 scanner.btc: Counted deposits from block hash="000000000000018d8ece83a004c5a919210d67798d13aa901c4d07f8bf87b719" height=235205 scannedDeposits=31
[2017-11-01T04:47:49Z] INFO btc.go:179 scanner.btc: Scanned deposits from block bestHeight=235214 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:0} hash="000000000000018d8ece83a004c5a919210d67798d13aa901c4d07f8bf87b719" height=235205 scannedDeposits=31 totalScannedDeposits=31
[2017-11-01T04:47:49Z] DEBUG btc.go:358 scanner.btc: Waiting for the next block blockHash="000000000000018d8ece83a004c5a919210d67798d13aa901c4d07f8bf87b719" blockHeight=235205
[2017-11-01T04:47:49Z] INFO btc.go:257 scanner.btc: Sending deposit to depositC deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:2952696 Height:235205 Tx:2bc199bd35e3e49cac62cb4b96357a7570ed183fbbfe5f3fae28a40785c86032 N:1 Processed:false}
[2017-11-01T04:47:49Z] DEBUG btc.go:296 scanner.btc: Scanning block hash="00000000000001588e7832fd94d683e477a9bb25c10921d25749f93c265493f6" height=235206
[2017-11-01T04:47:49Z] INFO btc.go:275 scanner.btc: Deposit is processed deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:2952696 Height:235205 Tx:2bc199bd35e3e49cac62cb4b96357a7570ed183fbbfe5f3fae28a40785c86032 N:1 Processed:false}
[2017-11-01T04:47:49Z] INFO btc.go:257 scanner.btc: Sending deposit to depositC deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:5430 Height:235205 Tx:c15c075a3a2487c39952b6f96f694dc679b1a5922c6d483ac0eea18a968b33c4 N:0 Processed:false}
[2017-11-01T04:47:49Z] INFO btc.go:275 scanner.btc: Deposit is processed deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:5430 Height:235205 Tx:c15c075a3a2487c39952b6f96f694dc679b1a5922c6d483ac0eea18a968b33c4 N:0 Processed:false}
[2017-11-01T04:47:49Z] INFO btc.go:257 scanner.btc: Sending deposit to depositC deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:86350000 Height:235205 Tx:275a1bc55ebe718e1f8b1ba5e107898fb7d6eebbc557372a96c533258486cab0 N:1 Processed:false}
[2017-11-01T04:47:49Z] INFO btc.go:305 scanner.btc: Counted deposits from block hash="00000000000001588e7832fd94d683e477a9bb25c10921d25749f93c265493f6" height=235206 scannedDeposits=47
[2017-11-01T04:47:49Z] INFO btc.go:179 scanner.btc: Scanned deposits from block bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="00000000000001588e7832fd94d683e477a9bb25c10921d25749f93c265493f6" height=235206 scannedDeposits=47 totalScannedDeposits=78
[2017-11-01T04:47:49Z] DEBUG btc.go:358 scanner.btc: Waiting for the next block blockHash="00000000000001588e7832fd94d683e477a9bb25c10921d25749f93c265493f6" blockHeight=235206
[2017-11-01T04:47:49Z] INFO btc.go:275 scanner.btc: Deposit is processed deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:5430 Height:235205 Tx:6917609f808ee48eb34fc3cd65289b00cb4f258daa2c6cfcdb7953ed2dff712f N:0 Processed:false}
[2017-11-01T04:47:49Z] INFO btc.go:257 scanner.btc: Sending deposit to depositC deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:1902696 Height:235205 Tx:1c200c69b9b16d5b36021c62c3d5ddffba8499a08fd135af4eacc312493327a3 N:1 Processed:false}
[2017-11-01T04:47:49Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:49Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:49Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:49Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:49Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:49Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:49Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:49Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:49Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:49Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:49Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:49Z] INFO btc.go:275 scanner.btc: Deposit is processed deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:1902696 Height:235205 Tx:1c200c69b9b16d5b36021c62c3d5ddffba8499a08fd135af4eacc312493327a3 N:1 Processed:false}
[2017-11-01T04:47:49Z] INFO btc.go:257 scanner.btc: Sending deposit to depositC deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:5430 Height:235205 Tx:8f6ed1882559d4a66ffe8a0c84f81ca185890935b70ad71580516270762fa7d7 N:0 Processed:false}
[2017-11-01T04:47:49Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:49Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:49Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:49Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:49Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:49Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:49Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:49Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:49Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:49Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:49Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:305 scanner.btc: Counted deposits from block hash="00000000000001588e7832fd94d683e477a9bb25c10921d25749f93c265493f6" height=235206 scannedDeposits=48
[2017-11-01T04:47:50Z] INFO btc.go:179 scanner.btc: Scanned deposits from block bestHeight=235214 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:0} hash="00000000000001588e7832fd94d683e477a9bb25c10921d25749f93c265493f6" height=235206 scannedDeposits=48 totalScannedDeposits=79
[2017-11-01T04:47:50Z] DEBUG btc.go:358 scanner.btc: Waiting for the next block blockHash="00000000000001588e7832fd94d683e477a9bb25c10921d25749f93c265493f6" blockHeight=235206
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] DEBUG btc.go:296 scanner.btc: Scanning block hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:275 scanner.btc: Deposit is processed deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:5430 Height:235205 Tx:8f6ed1882559d4a66ffe8a0c84f81ca185890935b70ad71580516270762fa7d7 N:0 Processed:false}
[2017-11-01T04:47:50Z] INFO btc.go:257 scanner.btc: Sending deposit to depositC deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:85300000 Height:235205 Tx:a8c8fcbd9ced75d9e1a0edc99f7cd02cb6016f805fb3d8d77232dc389103c9ac N:1 Processed:false}
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:275 scanner.btc: Deposit is processed deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:86350000 Height:235205 Tx:275a1bc55ebe718e1f8b1ba5e107898fb7d6eebbc557372a96c533258486cab0 N:1 Processed:false}
[2017-11-01T04:47:50Z] INFO btc.go:257 scanner.btc: Sending deposit to depositC deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:5430 Height:235205 Tx:6917609f808ee48eb34fc3cd65289b00cb4f258daa2c6cfcdb7953ed2dff712f N:0 Processed:false}
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:275 scanner.btc: Deposit is processed deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:85300000 Height:235205 Tx:a8c8fcbd9ced75d9e1a0edc99f7cd02cb6016f805fb3d8d77232dc389103c9ac N:1 Processed:false}
[2017-11-01T04:47:50Z] INFO btc.go:257 scanner.btc: Sending deposit to depositC deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:5430 Height:235205 Tx:addb4f5046726039314105b8eb7399e7ae3191b5c6e88d94d3cb5df16f17d0ae N:0 Processed:false}
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:275 scanner.btc: Deposit is processed deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:5430 Height:235205 Tx:6917609f808ee48eb34fc3cd65289b00cb4f258daa2c6cfcdb7953ed2dff712f N:0 Processed:false}
[2017-11-01T04:47:50Z] INFO btc.go:257 scanner.btc: Sending deposit to depositC deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:1902696 Height:235205 Tx:1c200c69b9b16d5b36021c62c3d5ddffba8499a08fd135af4eacc312493327a3 N:1 Processed:false}
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:275 scanner.btc: Deposit is processed deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:5430 Height:235205 Tx:addb4f5046726039314105b8eb7399e7ae3191b5c6e88d94d3cb5df16f17d0ae N:0 Processed:false}
[2017-11-01T04:47:50Z] INFO btc.go:257 scanner.btc: Sending deposit to depositC deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:84250000 Height:235205 Tx:932d25640734da305b3dc990cc183a4022ef33b3e37e3aabfa5dd9bbfff51cb6 N:1 Processed:false}
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:275 scanner.btc: Deposit is processed deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:84250000 Height:235205 Tx:932d25640734da305b3dc990cc183a4022ef33b3e37e3aabfa5dd9bbfff51cb6 N:1 Processed:false}
[2017-11-01T04:47:50Z] INFO btc.go:257 scanner.btc: Sending deposit to depositC deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:5430 Height:235205 Tx:c0e2b41065bcb4d788b1a2614a1dd0d0d9645412c01148ef90a4761250e79dab N:0 Processed:false}
[2017-11-01T04:47:50Z] INFO btc.go:305 scanner.btc: Counted deposits from block hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207 scannedDeposits=23
[2017-11-01T04:47:50Z] INFO btc.go:179 scanner.btc: Scanned deposits from block bestHeight=235214 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:0} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207 scannedDeposits=23 totalScannedDeposits=102
[2017-11-01T04:47:50Z] DEBUG btc.go:358 scanner.btc: Waiting for the next block blockHash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" blockHeight=235207
[2017-11-01T04:47:50Z] INFO btc.go:275 scanner.btc: Deposit is processed deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:5430 Height:235205 Tx:c0e2b41065bcb4d788b1a2614a1dd0d0d9645412c01148ef90a4761250e79dab N:0 Processed:false}
[2017-11-01T04:47:50Z] INFO btc.go:257 scanner.btc: Sending deposit to depositC deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:852696 Height:235205 Tx:fe96654b004d808350de6ed6b97ab954430f8f1ad1ccde4dd766978ce833d929 N:1 Processed:false}
[2017-11-01T04:47:50Z] INFO btc.go:275 scanner.btc: Deposit is processed deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:1902696 Height:235205 Tx:1c200c69b9b16d5b36021c62c3d5ddffba8499a08fd135af4eacc312493327a3 N:1 Processed:false}
[2017-11-01T04:47:50Z] INFO btc.go:257 scanner.btc: Sending deposit to depositC deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:5430 Height:235205 Tx:8f6ed1882559d4a66ffe8a0c84f81ca185890935b70ad71580516270762fa7d7 N:0 Processed:false}
[2017-11-01T04:47:50Z] INFO btc.go:275 scanner.btc: Deposit is processed deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:852696 Height:235205 Tx:fe96654b004d808350de6ed6b97ab954430f8f1ad1ccde4dd766978ce833d929 N:1 Processed:false}
[2017-11-01T04:47:50Z] INFO btc.go:257 scanner.btc: Sending deposit to depositC deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:5430 Height:235205 Tx:8a3395a99189f5d750bc3e7732b53098905b69f6eeb288687f5395f30611b10c N:0 Processed:false}
[2017-11-01T04:47:50Z] INFO btc.go:275 scanner.btc: Deposit is processed deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:5430 Height:235205 Tx:8f6ed1882559d4a66ffe8a0c84f81ca185890935b70ad71580516270762fa7d7 N:0 Processed:false}
[2017-11-01T04:47:50Z] INFO btc.go:257 scanner.btc: Sending deposit to depositC deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:85300000 Height:235205 Tx:a8c8fcbd9ced75d9e1a0edc99f7cd02cb6016f805fb3d8d77232dc389103c9ac N:1 Processed:false}
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:275 scanner.btc: Deposit is processed deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:5430 Height:235205 Tx:8a3395a99189f5d750bc3e7732b53098905b69f6eeb288687f5395f30611b10c N:0 Processed:false}
[2017-11-01T04:47:50Z] INFO btc.go:257 scanner.btc: Sending deposit to depositC deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:83200000 Height:235205 Tx:33e84093ecc6ac707ecd3870ea8716d0fa2e0d4f95299cfb3142cd5ac6e22b71 N:1 Processed:false}
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:275 scanner.btc: Deposit is processed deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:85300000 Height:235205 Tx:a8c8fcbd9ced75d9e1a0edc99f7cd02cb6016f805fb3d8d77232dc389103c9ac N:1 Processed:false}
[2017-11-01T04:47:50Z] INFO btc.go:257 scanner.btc: Sending deposit to depositC deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:5430 Height:235205 Tx:addb4f5046726039314105b8eb7399e7ae3191b5c6e88d94d3cb5df16f17d0ae N:0 Processed:false}
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:275 scanner.btc: Deposit is processed deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:83200000 Height:235205 Tx:33e84093ecc6ac707ecd3870ea8716d0fa2e0d4f95299cfb3142cd5ac6e22b71 N:1 Processed:false}
[2017-11-01T04:47:50Z] INFO btc.go:257 scanner.btc: Sending deposit to depositC deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:82050000 Height:235205 Tx:3d84467ae605fa53d421acf1c4b9f0aebe755ff205d3d0617e1e5d1f05e534e1 N:1 Processed:false}
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] DEBUG btc.go:296 scanner.btc: Scanning block hash="000000000000003c332218db768c18ef446f955f4637747badd1632c050d48c4" height=235208
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:275 scanner.btc: Deposit is processed deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:5430 Height:235205 Tx:addb4f5046726039314105b8eb7399e7ae3191b5c6e88d94d3cb5df16f17d0ae N:0 Processed:false}
[2017-11-01T04:47:50Z] INFO btc.go:257 scanner.btc: Sending deposit to depositC deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:84250000 Height:235205 Tx:932d25640734da305b3dc990cc183a4022ef33b3e37e3aabfa5dd9bbfff51cb6 N:1 Processed:false}
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:275 scanner.btc: Deposit is processed deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:82050000 Height:235205 Tx:3d84467ae605fa53d421acf1c4b9f0aebe755ff205d3d0617e1e5d1f05e534e1 N:1 Processed:false}
[2017-11-01T04:47:50Z] INFO btc.go:257 scanner.btc: Sending deposit to depositC deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:5430 Height:235205 Tx:e4b4fb39e52efeb16223ac79794708a8f6e3dfe99053bdf5a7e8b4e6a42e0ca6 N:0 Processed:false}
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:305 scanner.btc: Counted deposits from block hash="000000000000003c332218db768c18ef446f955f4637747badd1632c050d48c4" height=235208 scannedDeposits=0
[2017-11-01T04:47:50Z] INFO btc.go:179 scanner.btc: Scanned deposits from block bestHeight=235214 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:0} hash="000000000000003c332218db768c18ef446f955f4637747badd1632c050d48c4" height=235208 scannedDeposits=0 totalScannedDeposits=102
[2017-11-01T04:47:50Z] DEBUG btc.go:358 scanner.btc: Waiting for the next block blockHash="000000000000003c332218db768c18ef446f955f4637747badd1632c050d48c4" blockHeight=235208
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] DEBUG btc.go:296 scanner.btc: Scanning block hash="000000000000016739b24632d80a3fb8d85b177ffbe130ad9d636f9e1de40d8c" height=235209
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:275 scanner.btc: Deposit is processed deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:5430 Height:235205 Tx:e4b4fb39e52efeb16223ac79794708a8f6e3dfe99053bdf5a7e8b4e6a42e0ca6 N:0 Processed:false}
[2017-11-01T04:47:50Z] INFO btc.go:257 scanner.btc: Sending deposit to depositC deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:5430 Height:235205 Tx:03eaa315800a31edbe33cbd9b61e9147a96aa48c688e69a5d4595d2b4537f39a N:0 Processed:false}
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:275 scanner.btc: Deposit is processed deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:84250000 Height:235205 Tx:932d25640734da305b3dc990cc183a4022ef33b3e37e3aabfa5dd9bbfff51cb6 N:1 Processed:false}
[2017-11-01T04:47:50Z] INFO btc.go:257 scanner.btc: Sending deposit to depositC deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:5430 Height:235205 Tx:c0e2b41065bcb4d788b1a2614a1dd0d0d9645412c01148ef90a4761250e79dab N:0 Processed:false}
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:275 scanner.btc: Deposit is processed deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:5430 Height:235205 Tx:03eaa315800a31edbe33cbd9b61e9147a96aa48c688e69a5d4595d2b4537f39a N:0 Processed:false}
[2017-11-01T04:47:50Z] INFO btc.go:257 scanner.btc: Sending deposit to depositC deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:81000000 Height:235205 Tx:6bac487af445f17fc76b8cf5902693dc05f7ff817b9e6db50afa73afd18271e6 N:1 Processed:false}
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:275 scanner.btc: Deposit is processed deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:5430 Height:235205 Tx:c0e2b41065bcb4d788b1a2614a1dd0d0d9645412c01148ef90a4761250e79dab N:0 Processed:false}
[2017-11-01T04:47:50Z] INFO btc.go:257 scanner.btc: Sending deposit to depositC deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:852696 Height:235205 Tx:fe96654b004d808350de6ed6b97ab954430f8f1ad1ccde4dd766978ce833d929 N:1 Processed:false}
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:50Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:51Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:51Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:51Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:51Z] INFO btc.go:152 scanner.btc: Not enough confirmations, waiting bestHeight=235208 config={ScanPeriod:10ms DepositBufferSize:5 InitialScanHeight:235205 ConfirmationsRequired:2} hash="000000000000014e5217c81d6228a9274395a8bee3eb87277dd9e4315ee0f439" height=235207
[2017-11-01T04:47:51Z] INFO btc.go:216 scanner.btc: Closing BTC scanner
[2017-11-01T04:47:51Z] INFO btc.go:220 scanner.btc: Waiting for BTC scanner to stop
[2017-11-01T04:47:51Z] INFO btc.go:275 scanner.btc: Deposit is processed deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:81000000 Height:235205 Tx:6bac487af445f17fc76b8cf5902693dc05f7ff817b9e6db50afa73afd18271e6 N:1 Processed:false}
[2017-11-01T04:47:51Z] INFO btc.go:257 scanner.btc: Sending deposit to depositC deposit={CoinType:BTC Address:1LEkderht5M5yWj82M87bEd4XDBsczLkp9 Value:5430 Height:235205 Tx:a4d79bcc51cc34ab409b12833e94515a516b6340c049ccd78598b06674aef6a7 N:0 Processed:false}
panic: send on closed channel
goroutine 15 [running]:
github.com/skycoin/teller/src/scanner.(*BTCScanner).processDeposit(0xc4201d8150, 0x751826, 0x3, 0xc4203cc3c0, 0x22, 0x1536, 0x396c5, 0xc4203d4000, 0x40, 0x0, ...)
	/home/travis/gopath/src/github.com/skycoin/teller/src/scanner/btc.go:261 +0x34e
github.com/skycoin/teller/src/scanner.(*BTCScanner).Run.func3(0xc4201b4310, 0xc4201d8150)
	/home/travis/gopath/src/github.com/skycoin/teller/src/scanner/btc.go:197 +0x1e4
created by github.com/skycoin/teller/src/scanner.(*BTCScanner).Run
	/home/travis/gopath/src/github.com/skycoin/teller/src/scanner/btc.go:190 +0x5a0
FAIL	github.com/skycoin/teller/src/scanner	2.352s
ok  	github.com/skycoin/teller/src/sender	6.013s
ok  	github.com/skycoin/teller/src/teller	0.014s [no tests to run]
?   	github.com/skycoin/teller/src/util/dbutil	[no test files]
?   	github.com/skycoin/teller/src/util/httputil	[no test files]
ok  	github.com/skycoin/teller/src/util/logger	0.003s
ok  	github.com/skycoin/teller/src/util/mathutil	0.004s
?   	github.com/skycoin/teller/src/util/testutil	[no test files]
```

--- a/src/scanner/btc.go
+++ b/src/scanner/btc.go
@@ -214,8 +214,8 @@ func (s *BTCScanner) Run() error {
 // Shutdown shutdown the scanner
 func (s *BTCScanner) Shutdown() {
 	s.log.Info("Closing BTC scanner")
-	close(s.depositC)
 	close(s.quit)
+	close(s.depositC)
 	s.btcClient.Shutdown()
 	s.log.Info("Waiting for BTC scanner to stop")
 	<-s.done

